uk/lessons/basics/collections.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.2.1
+version: 1.0.0
```

Comment:
I think that the translation version should be 1.2.1 since its contents is up-to-date with the latest English lesson.

Reply:
Yep, thanks. I'll fix that. Just a moment.

Reply:
Fixed! @nscyclone please re-check it again. Thanks!